doc/developer/setup/index.md

```diff
@@ -143,6 +175,14 @@ To run the full integration test suite for IRIDA please run the following:
 
 This will run all the integration test profiles using Gradle, and print out reports for each profile.
 
+#### Memory requirements
+
+Your environment may throw java heap size errors when running tests. 
+
+You can increase the heap size by editing the `gradle.properties` file in the project directory and adding the following line:
+
+`org.gradle.jvmargs=-Xmx4096m`
```

Comment:
This can also be done with `env GRADLE_OPTS='-Xmx4096m'` this before running `./gradlew` commands. Though that would likely not apply to your IDE. One thing to note that after adding this option I have had to call `./gradlew --stop` to kill the active daemons to actually apply the memory change when re-launching gradle.

Reply:
I added some text for that method to the docs